server/controllers/common.js

```diff
@@ -21,6 +21,42 @@ router.get('/robots.txt', (req, res, next) => {
   }
 })
 
+/**
+ * Sitemap.xml
+ */
+router.get('/sitemap.xml', async function (req, res) {
+  !WIKI.config.seo.sitemap && res.status(404).end()
+
+  const host = WIKI.config.host
+  const xmlTree = [
+    '<?xml version="1.0" encoding="UTF-8"?>',
+    '<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">'
+  ]
+
+  try {
+    const pages = await WIKI.models.pages.query()
+      .select(['path', 'localeCode', 'updatedAt'])
+      .where('isPublished', '=', true)
+      .where('isPrivate', '=', false)
+
+    pages.forEach(page => {
+      // Fix checkAccess attr
+      page.locale = page.localeCode
+
+      if (!WIKI.auth.checkAccess(req.user, ['read:pages'], page)) return
```

Comment:
Use `WIKI.auth.guest` instead of `req.user` to provide a consistent output to all users.

Reply:
Indeed, now is no difference between logged and unlogged users.